gulp/tasks/publish-bower.js

```diff
@@ -0,0 +1,35 @@
+var gulp = require('gulp');
+var awspublish = require('gulp-awspublish');
+ 
+gulp.task('publish-bower', ['production'], function() {
```

Comment:
the name of the task in incorrect, it does not publish in bower, maybe publish static? publish browser library? upload library?